### PR TITLE
Fix a bug parsing kubeconfig if the user context was empty string.

### PR DESCRIPTION
**Pull request by brendandburns**
@drubin

### CODE REVIEW

File: kubernetes/src/kubernetes/config/kube_config.rb [hunk 1]
```diff
@@ -146,7 +146,9 @@ def find_context(name)
         if context['cluster']
           context['cluster'] = find_cluster(context['cluster'])
         end
-        context['user'] = find_user(context['user']) if context['user']
+        if context['user'] && context['user'].length > 0
```
Comment (drubin) [inline kubernetes/src/kubernetes/config/kube_config.rb]: Just FYI you can do `context['user'] = find_user(context['user']) if context['user'].blank?`  Then you should be able to keep the line the same.

https://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails 

I don't think it matters too much but if you do keep the `if` block can we remove the 2nd if check its kinda redundant and makes it harder to read.

Comment (drubin) [reply kubernetes/src/kubernetes/config/kube_config.rb]: Also, I think it would solve the rubocop warnings too :)

**Comment by brendandburns** [comment]
@drubin looks like `blank?` is in Rails, but not core Ruby, so we have to keep it like this for now...

(and using an `if` suffix here makes the line too long.

**Comment by brendandburns** [comment]
@drubin friendly ping on this one if you're around.

Thanks!

**Comment by drubin** [comment]
@brendandburns sorry been a busy week.  Also sorry about the `.blank?` being rails only I always forget what they add for you. 

/lgtm

**Pull request merged**